netbox/dcim/filtersets.py

```diff
@@ -909,6 +909,10 @@ class DeviceFilterSet(NetBoxModelFilterSet, TenancyFilterSet, ContactModelFilter
         queryset=Rack.objects.all(),
         label=_('Rack (ID)'),
     )
+    has_position = django_filters.BooleanFilter(
```

Comment:
We shouldn't need to introduce a new filter for this; the `position` filter should support a null (empty) lookup. I've opened #13585 to address this.